plugins/3MFReader/ThreeMFWorkspaceReader.py

```diff
@@ -926,7 +926,7 @@ def _applyMaterials(self, global_stack, extruder_stack_dict):
             build_plate_id = global_stack.variant.getId()
 
             # get material diameter of this extruder
-            machine_material_diameter = extruder_stack.materialDiameter
+            machine_material_diameter = extruder_stack.comptabileMaterialDiameter
```

Comment:
Use the getter function.
```suggestion
            machine_material_diameter = extruder_stack.getComptabileMaterialDiameter()
```